app/controllers/api/account.php

```diff
@@ -3958,12 +3958,11 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
     ->label('sdk.response.code', Response::STATUS_CODE_NOCONTENT)
     ->label('sdk.response.model', Response::MODEL_NONE)
     ->param('type', null, new WhiteList([Type::TOTP]), 'Type of authenticator.')
-    ->param('otp', '', new Text(256), 'Valid verification token.')
```

Comment:
We'll need a request filter for this removal

Reply:
Noted :+1: